packages/market-service/src/osmosis/utils.ts

```diff
@@ -0,0 +1,114 @@
+import { AssetReference } from '@shapeshiftoss/caip'
+import axios from 'axios'
+
+import { OsmosisMarketData, OsmosisPool } from './osmosis-types'
+
+const OSMOSIS_LCD_BASE_URL = 'https://daemon.osmosis.shapeshift.com/lcd/osmosis/'
+const OSMOSIS_IMPERATOR_BASE_URL = 'https://api-osmosis.imperator.co/'
```

Comment:
Fine as is since this is highly isolated to Osmosis market-service, but prefer passing these from web in `MarketServiceManagerArgs` for consistency and discoverability - if we ever need to change these, we can easily do so from web and without a lib bump as an added bonus

Reply:
Agreed. I fixed this in the latest round of changes.